dom.bs

```diff
@@ -210,8 +210,17 @@ run these steps:
 
  <li><p>Let <var>localName</var> be <var>qualifiedName</var>.
 
- <li><p>If <var>qualifiedName</var> contains a U+003A (:), then <a>strictly split</a> the string on
- it and set <var>prefix</var> to the part before and <var>localName</var> to the part after.
+ <li>
+  <p>If <var>qualifiedName</var> contains a U+003A (:), then:
+
+  <ol>
+   <li><p>Let <var>splitResult</var> be the result of <a>strictly split</a> given
```

Comment:
I think it should be either "of running strictly split" or "of strictly splitting" (this will link correctly automatically, in case you're wondering).

Reply:
> "of strictly splitting" (this will link correctly automatically, in case you're wondering).

That's really neat! I did not expect that to link correctly